cmd/crane/cmd/mutate.go

```diff
@@ -118,22 +117,21 @@ func NewCmdMutate(options *[]crane.Option) *cobra.Command {
 			fmt.Println(r.Context().Digest(digest.String()))
 		},
 	}
-	mutateCmd.Flags().StringSliceVarP(&anntns, "annotation", "a", nil, "New annotations to add")
-	mutateCmd.Flags().StringSliceVarP(&lbls, "label", "l", nil, "New labels to add")
-	mutateCmd.Flags().StringVar(&entrypoint, "entrypoint", "", "New entrypoint to set")
+	mutateCmd.Flags().StringToStringVarP(&anntns, "annotation", "a", nil, "New annotations to add")
+	mutateCmd.Flags().StringToStringVarP(&lbls, "label", "l", nil, "New labels to add")
+	mutateCmd.Flags().StringVar(&entrypoint, "entrypoint", "", "New entrypoing to set")
 	mutateCmd.Flags().StringVarP(&newRef, "tag", "t", "", "New tag to apply to mutated image. If not provided, push by digest to the original image repository.")
 	return mutateCmd
 }
 
-// splitKeyVals splits key value pairs which is in form hello=world
-func splitKeyVals(kvPairs []string) (map[string]string, error) {
+// validateKeyVals ensures no values are empty
+func validateKeyVals(kvPairs map[string]string) (map[string]string, error) {
```

Comment:
This could just return `error`, and not bother creating a copy of the input kv map.

Reply:
All set! I like that much better - we can use "labels" and "annotations" off the bat.